pkg/oc/originpolymorphichelpers/logsforobject.go

```diff
@@ -77,9 +77,6 @@ func NewLogsForObjectFn(delegate polymorphichelpers.LogsForObjectFunc) polymorph
 			if !ok {
 				return nil, errors.New("provided options object is not a v1.BuildLogOptions")
 			}
-			if bopts.Version != nil {
```

Comment:
i believe this was intended to handle the case of "oc logs build/test-1 --version=1" (you can't specify the version of the build and the build itself as build itself does not have versions ;-)

Reply:
so I would maybe keep this just distinguish between BuildConfig and Build

Reply:
oh, and now I expanded the diff and see that you already doing it ;-) deleting myself.